runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -181,6 +186,11 @@
 	private LoadingCache<NamedQuantity, Boolean> hiddenItems;
 	private final Queue<Integer> droppedItemQueue = EvictingQueue.create(16); // recently dropped items
 
+	@Getter
+	private final Map<Integer, GroundItem> memorizedGroundItemsToDisplay = new LinkedHashMap<>();
+	private final Map<GroundItem.GroundItemKey, Integer> despawnedGroundItems = new LinkedHashMap<>();
```

Comment:
Is there some known upper limit of item respawn lengths? If so, we should clear despawns which are older than that length.

Of the items on this page: https://oldschool.runescape.wiki/w/Item_spawn, I don't see many which would have very long spawn times. Maybe some of the wilderness items take the longest to respawn? We could also check with the wiki folk or shoot Mod Ash a tweet and ask about it.